packages/manager/modules/pci/src/components/hds/controller.js

```diff
@@ -0,0 +1,16 @@
+import { Environment } from '@ovh-ux/manager-config';
```

Comment:
**suggestion**:
Instead of importing the Environment module here, to get the user, why not passing the user through component binding ?

If you take a look here, the user is defined https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/index.js#L86, so you could use it

Reply:
thanks @JeremyDec, i preferred to import it directly for 3 reasons:
1 - because for me the '@ovh-ux/manager-config' it's a module utility (to share easily some reusable functions)
2 - if i want to use the 'me' variable i must to propagate me across all states until the HDS component which is very heavy.
3 - HDS it's a component, so i developed it to avoid hard links (couplage fort).

Reply:
@mohammed-zahaf in response of your reasons:
1 - Yes it is a module utility, which is already used in the root state resolve. Using the module in your component, implies to duplicate logic already existing elsewhere.
2 - In fact, as the user variable `me` is fetched in 'pci' parent state, you would just have to retrieve the user in the concerned state (by passing it in the component bindings).
3 - In terms of hard links, you're creating a dependency between the utility module and the component, which is not relevant, IMO in that case.

Reply:
I think it's one another way to do the stuff.
From my side, because HDS is a component without a state, i preferred to import Environment instead to propgate `me` until the states where the HDS is implemented and pass it as attribute for HDS component.

Reply:
Others pci component use the same strategy:
+ packages/manager/modules/pci/src/components
+ packages/manager/modules/pci/src/components/project/billing/volume-list/controller.js
+ packages/manager/modules/pci/src/components/project/billing/snapshot-list/controller.js